operators/src/engine/operator.rs

```diff
@@ -65,28 +65,28 @@ pub trait InitializedOperator {
 }
 
 pub type InitializedVectorOperator =
-    dyn InitializedOperatorB<VectorResultDescriptor, TypedVectorQueryProcessor>;
+    dyn InitializedOperator<VectorResultDescriptor, TypedVectorQueryProcessor>;
 
 pub type InitializedRasterOperator =
-    dyn InitializedOperatorB<RasterResultDescriptor, TypedRasterQueryProcessor>;
+    dyn InitializedOperator<RasterResultDescriptor, TypedRasterQueryProcessor>;
 
-pub trait InitializedOperatorB<R, Q>: InitializedOperator<Descriptor = R> + Send + Sync
+pub trait InitializedOperator<R, Q>: InitializedOperatorBase<Descriptor = R> + Send + Sync
 where
     R: ResultDescriptor + std::clone::Clone,
```

Comment:
ResultDescriptor should required `Clone` instead of requiring it here I guess.

Reply:
51e1381cde7d2c2f6080c710fb06190362843ef2

Reply:
Does `map` on a result descriptor make sense if it is not required to be clonable?

Reply:
Oh okay, ist `Copy`